src/cl.rs

```diff
@@ -127,6 +130,23 @@ pub(crate) enum Action {
         /// Target directory (must exist)
         dir: PathBuf,
     },
+    #[cfg(feature = "gen-man-pages")]
+    GenShellComplete {
+        #[clap(flatten)]
+        arg_group: GenShellCompleteGroup,
+    },
+}
+
+#[cfg(feature = "gen-man-pages")]
+#[derive(Debug, clap::Args)]
+#[group(required = true, multiple = true)]
+pub(crate) struct GenShellCompleteGroup {
+    /// Shell to generate for, leave empty for all
+    #[arg(short = 's', long, default_value = None)]
+    pub shell: Option<Shell>,
+    /// Target directory
+    #[arg(num_args = 1, required = false)]
```

Comment:
I think you can remove this line, this is already the default behavior.

Reply:
Could you elaborate on "this" line

Reply:
GitHub is not great for reviews, but my comment was on line 148.